src/style.css

```diff
@@ -1 +1,63 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
```

Comment:
it's better not set uppercase as a default value for all of your elements. Use only at appropriate places